src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionTargetMFDs.java

```diff
@@ -136,7 +136,7 @@ public NZSHM22_SubductionInversionTargetMFDs(NZSHM22_InversionFaultSystemRuptSet
 //		List<MFD_InversionConstraint> mfdUncertaintyConstraints = new ArrayList<>();
 
 		if (mfdUncertaintyWeightedConstraintWt > 0.0) {
-			mfdUncertaintyConstraints.add(MFDManipulation.addMfdUncertainty(targetOnFaultSupraSeisMFD, MINIMIZE_RATE_BELOW_MAG, mfdUncertaintyWeightedConstraintPower));
+			mfdUncertaintyConstraints.add(MFDManipulation.addMfdUncertainty(targetOnFaultSupraSeisMFD, MINIMIZE_RATE_BELOW_MAG, 20, mfdUncertaintyWeightedConstraintPower));
```

Comment:
@voj wondering about hard coded 20

Reply:
hardcoded to affect no change to before. My plan is to see if this change is what works for the scientists before fixing subduction